scripts/release-binary.sh

```diff
@@ -4,12 +4,12 @@ project=$(basename $(pwd))
 
 TEMPDIR=`mktemp -d`
 
-REV=`git rev-parse HEAD`
-
 nix build -f release -o $TEMPDIR/$project
 
 tar --owner=serokell:1000 --mode='u+rwX' -czhf $TEMPDIR/release.tar.gz -C $TEMPDIR $project
 
 hub release delete auto-release
 
+git tag -d auto-release
```

Comment:
AFAIU, it will only delete the tag from your local repository. You should push tag removal to remote.

Reply:
Hmm, I think it'll automatically create&push the tag with `hub release create`. I might be wrong.

Reply:
```
       create
           Create a GitHub release for the specified TAG name. If git tag TAG does not exist, it will be created at TARGET (default: current branch).
```

Reply:
Anyway, it seems to be fixed.